modules/core/include/opencv2/core/mat.hpp

```diff
@@ -2408,7 +2419,7 @@ class CV_EXPORTS UMat
     //! assignment operators
     UMat& operator = (const UMat& m);
 
-    Mat getMat(int flags) const;
+    CV_WRAP_AS(get) Mat getMat(int flags CV_WRAP_DEFAULT(ACCESS_RW)) const;
 
```

Comment:
The interface is not changes, as `CV_WRAP_DEFAULT` affects only the wrapper. We can also utilize `CV_DEFAULT` instead to affect both the wrapper and the C++ interface, without affecting the C interface.